packaging/homebrew/shopify-cli.base.rb

```diff
@@ -0,0 +1,37 @@
+class ShopifyCli < Formula
+  version 'SHOPIFY_CLI_VERSION'
+  homepage 'https://shopify.github.io/shopify-app-cli'
+  url "https://rubygems.org/downloads/shopify-cli-#{version}.gem"
+  sha256 'SHOPIFY_CLI_GEM_CHECKSUM'
+  desc <<~DESC
+    Shopify CLI helps you build Shopify apps faster. It quickly scaffolds Node.js
+    and Ruby-on-Rails embedded apps. It also automates many common tasks in the
+    development process and lets you quickly add popular features, such as billing
+    and webhooks.
+  DESC
+
+  bottle :unneeded
+
+  def install
+    system 'tar', '-xzf', cached_download, '--directory', buildpath
+
+    (buildpath/'src').mkpath
+    (buildpath/'symlink').mkpath
+    system 'tar', '-xzf', buildpath/'data.tar.gz', '--directory', buildpath/'src'
```

Comment:
This is hacking the system a bit by leveraging the fact that the gem itself is a tarball, so I manually break it apart and take the actual source code to install it into brew.

I'd be curious to know if anyone thinks this is a bad idea.